examples/non-html-pages/src/pages/company.json.ts

```diff
@@ -0,0 +1,9 @@
+export async function get() {
```

Comment:
I'm not sure why this file is named `company.json.ts`, can we rename it to `about.json.ts` instead?

Reply:
<img width="971" alt="image" src="https://user-images.githubusercontent.com/990216/155210531-560af23d-8b4f-4eeb-8489-b2d5166984e6.png">


https://docs.astro.build/en/core-concepts/astro-pages/#non-html-pages

Reply:
doh! Mind changing in both places?

Reply:
I think company is fine; maybe ask whoever wrote the original example.